bin/run_celery_worker_log_parser

```diff
@@ -16,8 +16,8 @@ fi
 
 exec newrelic-admin run-program celery -A treeherder worker \
     -Q \
-parse_job_logs,\
-parse_job_logs_fail,\
+parse_job_log,\
+parse_job_log_fail,\
```

Comment:
Need to adjust `Procfile` too. Also, the old queue will end up with orphaned entries, so needs both old and new queue name to be listed initially, with the old queue removed in a followu-up.